express/blocks/search-marquee/search-marquee.js

```diff
@@ -0,0 +1,374 @@
+import { getLibs
+} from '../../scripts/utils.js';
+import { decorateButtonsDeprecated } from '../../scripts/utils/decorate.js';
+import {  getIconElement } from '../../scripts/utils/icons.js'; 
+import { addTempWrapperDeprecated } from '../../scripts/utils/decorate.js'
+import { buildFreePlanWidget } from '../../scripts/widgets/free-plan.js'
+import buildCarousel from '../shared/carousel.js';
+import fetchAllTemplatesMetadata from '../../scripts/utils/all-templates-metadata.js'
+import BlockMediator from '../../scripts/block-mediator.min.js';
+const {replaceKey} = await import(`${getLibs()}/features/placeholders.js`);
```

Comment:
You should use a promise.all here
by having an await on these three lines you're kicking off a request, and making everything wait until the response returns. Then you're going to the next line and doing the same thing. This is slow. 
Make the requests run in parallel.